core/common/library/Media/MediaFiles.php

```diff
@@ -12,8 +12,8 @@
  */
 namespace Phanbook\Media;
 
-use League\Flysystem\Filesystem;
-use League\Flysystem\Adapter\Local as Adapter;
+use \League\Flysystem\Filesystem;
+use \League\Flysystem\Adapter\Local as Adapter;
```

Comment:
Well, I think both have `\` and don't have `\` still ok. I add `\` because I get some bugs when I writing this feature and I add `\` like a test case.